### PR TITLE
use Environment.jruby?

### PR DESCRIPTION
**Pull request by colinsurprenant**
PR elastic/logstash#1242 rebranding

### CODE REVIEW

File: lib/logstash/outputs/file.rb [hunk 1]
```diff
@@ -220,7 +220,7 @@ def open(path)
 
     # work around a bug opening fifos (bug JRUBY-6280)
     stat = File.stat(path) rescue nil
-    if stat and stat.ftype == "fifo" and RUBY_PLATFORM == "java"
+    if stat && stat.ftype == "fifo" && LogStash::Environment.jruby?
       fd = java.io.FileWriter.new(java.io.File.new(path))
     else
```
Comment (ph) [inline lib/logstash/outputs/file.rb]: I saw what you have done here :speak_no_evil:

**Comment by ph** [comment]
lgtm

File: lib/logstash/outputs/file.rb (hunk 1 above)
Comment (colinsurprenant) [inline lib/logstash/outputs/file.rb]: :innocent:

Comment (suyograo) [inline lib/logstash/outputs/file.rb]: where's rubocop when you need him? :p

Comment (jordansissel) [inline lib/logstash/outputs/file.rb]: kitt

Comment (colinsurprenant) [inline lib/logstash/outputs/file.rb]: image

**Pull request merged**